roofit/histfactory/inc/RooStats/HistFactory/PiecewiseInterpolation.h

```diff
@@ -83,7 +83,7 @@ class PiecewiseInterpolation : public RooAbsReal {
     RooArgList _highIntList ;
     // will want std::vector<RooRealVar*> for low and high also
   } ;
-  mutable RooObjCacheManager _normIntMgr ; // The integration cache manager
+  mutable RooObjCacheManager _normIntMgr ; //! The integration cache manager
```

Comment:
Since you are editing the file to add the transient, I would use the correct syntax to have it visible in Doxygen. 
I think is : 
`///<`  for normal data mebers
`///<! ` for transient ones.

Reply:
Thanks for the hint! Thanks a good point, I'll try to think about this from now on when I change member variables next time.